smf/sled-agent/gimlet-standalone/config-rss.toml

```diff
@@ -104,7 +104,7 @@ uplink_port_speed = "40G"
 uplink_port_fec="none"
 # For softnpu, an address within the "infra" block above that will be used for
 # the softnpu uplink port.  You can just pick the first address in that pool.
-uplink_ip = "192.168.1.30"
+uplink_cidr = "192.168.1.30/32"
```

Comment:
I think these will parse out the same.

```
$ cat src/main.rs
fn main() {
    let v4_a: ipnetwork::Ipv4Network = "10.1.2.3".parse().unwrap();
    let v4_b: ipnetwork::Ipv4Network = "10.1.2.3/32".parse().unwrap();
    println!("{:#?}", v4_a);
    println!("{:#?}", v4_b);
}
```
```
$ cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.00s
     Running `target/debug/ipparse`
Ipv4Network {
    addr: 10.1.2.3,
    prefix: 32,
}
Ipv4Network {
    addr: 10.1.2.3,
    prefix: 32,
}
```

Reply:
You are correct! Given the name change to `..._cidr`, I thought it was slightly clearer to include the explicit `/32`, but if that is surprising or less clear we could take it back out.

Reply:
I'm going to go ahead and merge; happy to do a small followup to remove the `/32` from the config files if you'd prefer.